CHANGELOG

```diff
@@ -1,4 +1,30 @@
 # Changelog
+## [0.2.5] - 2020-02-19
+### Breaking changes
```

Comment:
This should also mention that NetworkManager 1.22.8 or later is now required.

Reply:
Thanks!